main.cc

```diff
@@ -966,6 +968,12 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 db.invoke_on_all(&replica::database::stop).get();
             });
 
+            task_manager.start(std::ref(*cfg), std::ref(stop_signal.as_sharded_abort_source())).get();
+            auto stop_task_manager = defer_verbose_shutdown("task_manager", [&task_manager] {
+                task_manager.invoke_on_all(&task_manager::stop).get();
```

Comment:
IIRC, this is called automatically by `sharded<task_manager>::stop()`, so no need to call it explicitly here.